src/pip/_internal/commands/install.py

```diff
@@ -362,8 +362,16 @@ def run(self, options, args):
                     )
                     resolver.resolve(requirement_set)
 
+                    try:
```

Comment:
```py
try:
    pip_req = requirement_set.get_requirements("pip")
except KeyError:
    modifying_pip = False
else:
    # If we're not replacing an already installed pip,
    # we're not modifying it.
    modifying_pip = pip_req.satisfied_by is None
```

Reply:
Thanks! Looks better now 👍

Reply:
Thanks for accepting my unintentionally-no-text-description suggestion! ^>^

Reply:
Sometimes the code says more that words :)